modules/ocf_mail/templates/site_vhost/mysql-alias-domains.erb

```diff
@@ -0,0 +1,8 @@
+hosts = mysql
+user = ocfmail-ro
+password = <%= @mysql_ro_password %>
+dbname = ocfmail
+query = SELECT domain FROM domains WHERE domain = '%s'
```

Comment:
`domains` is a view generated from the `addresses` table, so there is no duplication of data here